frontend/src/components/ClipboardCopy.vue

```diff
@@ -14,11 +14,11 @@
 export default {
   name: 'ClipboardCopy',
   props: {
-    code: { type: String, required: true },
+    link: { type: String, required: true },
   },
   data() {
     return {
-      url: `${window.location.origin}/redeem/${this.code}`,
+      url: `${this.link}`,
```

Comment:
rename field to link?

Reply:
Yes it is not a code anymore so changed the field code to link

Reply:
im talking about:

```suggestion
      link: `${this.link}`,
```

wouldnt it to make sense to use the same name - is this posisble?